test/unit/legacy_wrappers/gridfs.test.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const {
+  MongoClient,
+  GridFSBucket,
+  FindCursor,
+  GridFSBucketWriteStream
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { toLegacy } = require('../../../src/utils');
+
+const iLoveJs = 'mongodb://iLoveJavascript';
+
+describe('legacy_wrappers/gridfs.js', () => {
+  let client;
+  let bucket;
+  beforeEach(async function () {
+    client = new MongoClient(iLoveJs);
+    bucket = new GridFSBucket(client.db());
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it('should convert bucket.find(filter, options) to legacy version', () => {
+    const spy = sinon.spy(mongodbDriver.GridFSBucket.prototype, 'find');
```

Comment:
The signature for `find` is: `find(filter?: Filter<GridFSFile>, options?: FindOptions): FindCursor<GridFSFile>`.  Do we also need to cover the other overloads to ensure that they also convert the result of `bucket.find` to the legacy version?

Reply:
was this question answered?